src/shared/config/AuthXmlConfigFileLocator.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types = 1);
+namespace PharIo\Phive;
+
+class AuthXmlConfigFileLocator {
+    public const FILENAME = 'phive-auth.xml';
```

Comment:
Does this have to be public?

Reply:
Absolutely not. It's never used outside this class